lib/TPP/DefaultTppPasses.cpp

```diff
@@ -100,14 +101,25 @@ struct DefaultTppPasses
       pm.addNestedPass<func::FuncOp>(createLinalgLowering());
       pm.addPass(createCleanup());
     }
-
+    // Low level parallelization passes.
+    if (tileShapeM != 0 && tileShapeN != 0) {
+      LowLevelParallelizationOptions LowLevelParallelization(
+          LowLevelParallelizationOptions{tileShapeM, tileShapeN, shuffleOrder,
+                                         outerParallelLoops});
+      pm.addPass(createLowLevelParallelization(LowLevelParallelization));
+    }
     // Convert forAll to parallel loops should run after bufferization
     // as scf.parallel does not handle tensor.
     pm.addPass(createConvertForAllToParallelOp());
-
-    // Low leve parallelization passes.
-    LowLevelParallelizationOptions LowLevelParallelization{parallelTaskGrid};
-    pm.addPass(createLowLevelParallelization(LowLevelParallelization));
+    pm.addPass(createCombineXsmmOpPass());
```

Comment:
In a follow-up PR, this needs to be in a new bundle, both Xsmm lowering and AMX passes.
Please add a TODO.